examples/graphql-file-upload/graphql_file_upload.bal

```diff
@@ -0,0 +1,56 @@
+import ballerina/graphql;
+import ballerina/io;
+
+service /fileUpload on new graphql:Listener(4000) {
+
+    // Remote functions can use the `graphql:Upload` type as an input
+    // parameter type.
+    remote function singleFileUpload(graphql:Upload file)
+        returns string|error {
+
+        // Access the `file name` from `graphql:Upload` type parameter.
+        // Similarly, it can access the mime type as `file.mimeType`
+        // and encoding as `file.encoding`. Except the `byteStream` field, all
+        // other fields in the `graphql:Upload` are `string` values.
```

Comment:
```suggestion
        // Access the file name from the `graphql:Upload` type parameter.
        // Similarly, it can access the mime type as `file.mimeType`
        // and encoding as `file.encoding`. Except the `byteStream` field, all
        // other fields in the `graphql:Upload` are `string` values.
```